main/resource-access.bs

```diff
@@ -75,6 +75,291 @@ Issue: Write Linked Data Platform section.
+authority, per the configuration of the Solid server on which that resource
+resides.
+
+Metadata resources that reside on a Solid server MUST adhere to the same
+interaction model used by other standard Solid resources, except where specified
+in the definition of the associated metadata type in [[#rm-reserved]].
+
+A metadata resource MUST be deleted by the Solid server when the resource it is
+directly associated with is also deleted and the Solid server is authoritative
+for both resources.
+
+### Reserved Metadata Types ### {#rm-reserved}
+
+#### Web Access Control #### {#rm-wac}
+
+ACL resources as defined by [[!WAC]] MUST be supported as a resource metadata
+type by Solid servers.
+
+The ACL metadata resource directly associated with a given resource is
+discovered by the client via the `rel="acl"` `Link` relation in a `Link` header.
+
+A given Solid resource MUST NOT be directly associated with more than one ACL
+metadata resource. A given ACL metadata resource MUST NOT be directly associated
+with more than one Solid resource.
+
+To discover, read, create, or modify an ACL metadata resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Control](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+An ACL metadata resource MUST NOT be deleted unless the resource directly
+associated with it is deleted.
+
+A Solid server SHOULD sanity check ACL metadata resources upon creation or
+update to restrict invalid changes, such as by performing shape validation
+against authorization statements therein.
+
+#### Resource Description #### {#rm-description}
+
+Issue: Identify any resource description metadata parameters that MUST be
+recognized by all implementations.
+
+Resource description is a general mechanism to provide descriptive metadata for
+a given resource. It MUST be supported as a resource metadata type by Solid
+servers.
+
+The Descriptive metadata resource directly associated with a given resource is
+discovered by the client via the `"rel=describedby"` `Link` relation in a `Link`
+header. Conversely, the resource being described by a Descriptive metadata
+resource is discovered by the client via the `rel="describes"` `Link` relation
+in a `Link` header.
+
+A given Solid resource MUST NOT be directly associated with more than one
+Descriptive metadata resource.
+
+To create or modify a Descriptive metadata resource, a given
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Write](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
```

Comment:
Don't like; I think we should just let the normal ACL mechanism do its job and have it on the separate resource.

ACL should likely remain the only exception (because there's a very good reason).

Reply:
If we take this stance, then we are saying that effectively any client that wants to provide resource description needs control privileges, because the server won't be in a position to do it for them automatically. We should be doing as much as we can to ensure that basic operations can be done without control permissions, because otherwise we'll end up with basic clients that have much more privilege than they should. Issue created inline to track this.